polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -108,11 +113,12 @@ export function Item({
       aria-label={accessibilityLabel}
       onClick={disabled ? null : onAction}
       role={role}
+      ref={ref as Ref<HTMLAnchorElement>}
     >
       {contentElement}
     </UnstyledLink>
   ) : (
-    <button
+    <UnstyledButton
```

Comment:
This is a potentially breaking change; `UnstyledButton` resets a bunch of styles which a consumer may be relying on the browser implementing for them.

IMO this would warrant a `major` version bump for this PR (or, just revert back to `button`).

Reply:
i'll reset it and just pass ref as LegacyRef. thanks!